zh/deploy-on-gcp-gke.md

```diff
@@ -132,6 +132,8 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
 
         如上所述，生产环境的部署需要 91 个 CPU，超过了 GCP 项目的默认配额。可以参考[配额](https://cloud.google.com/compute/quotas)来增加项目配额。扩容同样需要更多 CPU。
 
+		- 默认创建的是 Regional 集群，会在 3 个可用区里都创建节点数量。比如配置 `pd_count = 1` ，实际为 PD 创建的节点数为 3 个。可以通过配置 `location` 来创建 Zonal 集群，具体可参见 `examples/` 下例子。
```

Comment:
```suggestion
		- 默认创建的是 Regional 集群，会在 3 个可用区里都创建节点数量。比如配置 `pd_count = 1`，实际为 PD 创建的节点数为 3 个。可以通过配置 `location` 来创建 Zonal 集群，具体可参见 `examples/` 下例子。
```

Reply:
Can we combine this with the note in L139?